src/block/defines.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I find it preferably to have the defines in corresponding files (as it was before). It is easier to find them if they're defined closer to where they're used or in the modules that they logically belong to.